src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerRemoveImage.groovy

```diff
@@ -15,10 +15,21 @@
  */
 package com.bmuschko.gradle.docker.tasks.image
 
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
 class DockerRemoveImage extends DockerExistingImage {
+
+    @Input
+    @Optional
+    Boolean force
+
     @Override
     void runRemoteCommand(dockerClient) {
         logger.quiet "Removing image with ID '${getImageId()}'."
-        dockerClient.removeImageCmd(getImageId()).exec()
+        def removeImageCmd = dockerClient.removeImageCmd(getImageId())
```

Comment:
@orzeh @mjacques lets stick with what we/mjacques did previously. It's more verbose but inline with what we do on other tasks and is very clear to the user what the intent is.